libwebrtc/src/native/audio_source.rs

```diff
@@ -78,8 +78,10 @@ impl NativeAudioSource {
                     interval.tick().await;
 
                     let inner = source.inner.lock().await;
-                    if inner.captured_frames > 0 {
-                        break; // User captured something, stop injecting silence
+                    if let Some(last_capture) = inner.last_capture {
+                        if last_capture.elapsed() < Duration::from_millis(20) {
```

Comment:
20ms instead of 10?
Not sure if this is the best way to handle it but it avoid to inject silence if the user is 1ms late

Reply:
20ms seems fine. we would inject 10ms worth of silence? 

we should have a break condition still somewhere right? when the source is destroyed?

Reply:
yes 10ms of silence every 10ms
blah I'm tired.. yeah I should stop the task

Reply:
I think it makes sense but I don't know enough about what this means for jitter buffer behavior. Basically every time you stop sending audio the pipeline will have 10ms of jitter introduced.